src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -13,40 +13,46 @@ import './DropdownMenu.scss';
 import classNames from 'classnames';
 
 export interface DropdownItem {
+  /** Unique ID to use in Array.map() */
   id: string;
+  /** Label to appear in the menu */
   label: string | ReactNode;
+  /** Callback to run when an item is selected */
   onSelect: (event?: Event) => void;
   className?: string;
-  icon?: string | ReactNode;
 }
 
 export interface DropdownMenuProps extends RadixUIDropdownMenuProps {
+  /** List of items to render in the dropdown menu */
   items: DropdownItem[];
-  trigger?: ReactNode | 'triple-dot';
+  /** Clicking this will make the dropdown menu appear */
+  trigger?: ReactNode;
 
   // from RadixDropdownMenu.DropdownMenuContentProps
   side?: 'top' | 'right' | 'bottom' | 'left';
   alignMenu?: 'start' | 'center' | 'end';
 
+  /** Class to apply to the trigger */
   className?: string;
-  menuClassName?: string;
 }
 
 export const DropdownMenu = forwardRef<HTMLDivElement, DropdownMenuProps>(
-  ({ className, alignMenu = 'center', items, trigger = 'triple-dot', side = 'bottom', ...dropdownMenuProps }, ref) => {
+  ({ className, alignMenu = 'center', items, trigger, side = 'bottom', ...dropdownMenuProps }, ref) => {
+    const triggerElement = !trigger ? <span>...</span> : typeof trigger === 'string' ? <span>{trigger}</span> : trigger;
+
     return (
       <RadixUIDropdownMenuRoot {...dropdownMenuProps}>
```

Comment:
as per our discussion in the chat room, let's not arbitrarily forward props. it's slightly more convenient initially, but the risks are just too great. aside from the risk factor. .. being explicit encourages thinking about what things are *actually* important to the current component, and can also be a signal when a component is getting too complex. it also helps reduce over-typing, and helps avoid tying your interface too tightly to another components implementation.

Reply:
On it.